package in.co.testException;

public class Test {
	
	public static void main(String[] args) {
		
		String s= "Bhag";
		String t= null;
		
		try{System.out.println(s.charAt(6));
		System.out.println(t.length());}
		
		catch(StringIndexOutOfBoundsException e) {
			System.out.println("String is small");
			/* System.exit(0); */
		}
		
		catch(NullPointerException p) {
			System.out.println("String is null");
		}
		
		finally {
			System.out.println("I am final");
		}
	}

}
